deps/wazuh_testing/wazuh_testing/modules/fim/__init__.py

```diff
@@ -178,6 +178,11 @@
 CB_FILE_EXCEEDS_DISK_QUOTA = r'.*The (.*) of the file size \'(.*)\' exceeds the disk_quota.*'
 CB_FILE_SIZE_LIMIT_REACHED = r'.*File \'(.*)\' is too big for configured maximum size to perform diff operation\.'
 CB_DIFF_FOLDER_DELETED = r'.*Folder \'(.*)\' has been deleted.*'
+CB_STARTING_WINDOWS_AUDIT = r'.*state_checker.*(Starting check of Windows Audit Policies and SACLs)'
+CB_SWITCHING_DIRECTORIES_TO_REALTIME = r'.*state_checker.*(Audit policy change detected.\
+                                         Switching directories to realtime)'

```

Comment:
Indentation is correct. Since the bottom line is part of the same regex string, it needs to start in line with the first character after the `r'`denoting the start of the string. In the code it is correct, but is shown differently in the web, based on resolution.